tests/test_resource_base.py

```diff
@@ -53,6 +52,19 @@
 if "ACTINIA_CUSTOM_TEST_CFG" in os.environ:
     custom_actinia_cfg = str(os.environ["ACTINIA_CUSTOM_TEST_CFG"])
 
+additional_external_data = {
+    "rio_json": "https://apps.mundialis.de/actinia_test_datasets/rio.json",
+    "brazil_json": "https://apps.mundialis.de/actinia_test_datasets/brazil_polygon.json",
+    "elev_ned_30m_tif": "https://apps.mundialis.de/actinia_test_datasets/elev_ned_30m.tif",
+    "elev_ned_30m_nope_tif": "https://apps.mundialis.de/actinia_test_datasets/elev_ned_30m_nope.tif",
+    "polygon_gml": "https://apps.mundialis.de/actinia_test_datasets/polygon.gml",
+    "census_wake2000_gml": "https://apps.mundialis.de/actinia_test_datasets/census_wake2000.gml",
+    "census_wake2000_zip": "https://apps.mundialis.de/actinia_test_datasets/census_wake2000.zip",
+    "geology_30m_tif": "https://apps.mundialis.de/actinia_test_datasets/geology_30m.tif",
+    "geology_30m_zip": "https://apps.mundialis.de/actinia_test_datasets/geology_30m.zip",
+    "pointInBonn": "https://raw.githubusercontent.com/mmacata/pagestest/gh-pages/pointInBonn.geojson"
```

Comment:
```suggestion
    "pointInBonn": "https://raw.githubusercontent.com/mmacata/pagestest/gh-pages/pointInBonn.geojson"
```

Would it be better to have all test data in a single place?